src/github.com/matrix-org/dendrite/common/config/config.go

```
@@ -360,6 +371,14 @@ func (config *Dendrite) derive() {
 		config.Derived.Registration.Flows = append(config.Derived.Registration.Flows,
 			authtypes.Flow{Stages: []authtypes.LoginType{authtypes.LoginTypeDummy}})
 	}
+
+	// Load application service configuration files
+	fmt.Println(config.ApplicationService.ConfigFiles)
+	if len(config.ApplicationService.ConfigFiles) > 0 {
+		if err := loadAppservices(config); err != nil {
+			fmt.Println("Error loading AS config: ", err)
```

should return an error rather than printing and continuing